docusaurus/website/src/pages/index.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import withBaseUrl from '@docusaurus/withBaseUrl';
+
+import Layout from '@theme/Layout';
+import CodeBlock from '@theme/CodeBlock';
+
+import classnames from 'classnames';
+
+import styles from './styles.module.css';
+
+const features = [
+  {
+    title: 'Less to Learn',
```

Comment:
I believe fragment syntax works here too: `<>...</>`. We could use fragments for better line wrapping.